src/app/activity/bounty/Create.ts

```diff
@@ -130,12 +132,23 @@ export const createBounty = async (createRequest: CreateRequest): Promise<any> =
             const dmChannel: DMChannel = await walletNeededMessage.channel.fetch() as DMChannel;
             
             await createRequest.commandContext.send({content: `Waiting for <@${owedTo.id}> to enter their wallet address.`, ephemeral: true});
```

Comment:
Yea, unfortunately I don't think there is another option

Since the only way to send an ephemeral message is in the context of an interaction, and the snowflake ID for that interaction message is created when the interaction is kicked off, that interaction message will always happen before a new message is created. In this case, the new message is the bounty card for the IOU